.github/workflows/soaking.yml

```diff
@@ -14,7 +14,7 @@
 name: 'weekly soaking test'
 on:
   schedule:
-    - cron: '0 10 * * SAT' # every Saturday at 10 am UTC: pst 2am
+    - cron: '0 10 * * SAT' # every Saturday at 10 am UTC: pst 2am'
```

Comment:
Looks like this added a typo?

Reply:
Another typo from me and fixed it. Thanks for the notice.